src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -114,6 +116,12 @@
       <PackageDestination Include="$(PackageTargetPath)">
         <TargetFramework Condition="'$(PackageTargetFramework)' != ''">$(PackageTargetFramework)</TargetFramework>
       </PackageDestination>
+      
+      <!-- RID-specific desktop libraries should also be packaged without a RID to work in packages.config projects -->
+      <PackageDestination Condition="'$(IsDesktopLibrary)' == 'true' AND '$(PackageTargetRuntime)' != '' AND '$(ExludeDesktopLibraryFromLib)' != 'true'" Include="lib/$(PackageTargetFramework)">
```

Comment:
Do we need the other ExludeDesktopLibraryFromLib property? If so I would rename Desktop similarly.